services/counting_repository.py

```
@@ -0,0 +1,226 @@
+"""CountingRepository: Consolidated counting minigame data access.
+
+Replaces 21 individual API functions in api.py with a single typed service.
+"""
+
+from enum import IntEnum
+from typing import Any
```

Flake8 (7.3.0)

[error] 'typing.Any' imported but unused

(F401)